Schedule.API/Services/Procedures/RecommendationService.cs

```diff
@@ -253,5 +254,94 @@ private IEnumerable<EquipmentRelocationDto> FindCommonEquipmentRelocationAppoint
             }
             return equipmentRelocationDtos;
         }
+
+        public IEnumerable<RoomRenovationDto> RecommendRenovationAppointments(SchedulingDto dto)
+        {
+            if(dto.SecondRoomId == -1)
+                return BasicRenovationSearch(dto);
+            else
+                return ComplexRenovationSearch(dto);            
+        }
+
+        private IEnumerable<RoomRenovationDto> BasicRenovationSearch(SchedulingDto dto)
+        {
+            List<RoomRenovationDto> appointments = new List<RoomRenovationDto>();
+            List<Shift> availableShiftsSource = new List<Shift>();
+            IEnumerable<Shift> shiftsForSourceRoom = _shiftWrapper.Repository.GetShiftsByRoomID(dto.FirstRoomId);
+            foreach (Shift shift in shiftsForSourceRoom)
+            {
+                List<Shift> shiftList = new List<Shift>();
+                shiftList.Add(shift);
+                IEnumerable<RecommendationDto> recommendationsSourceRoom = FindRecommendationsInShifts(shiftList, 16);
+                if (recommendationsSourceRoom.Count() == 16)
+                {
+                    availableShiftsSource.Add(shift);
+                }
```

Comment:
What number 16 should represent

Reply:
It represents maximum number of appointments in one shift. It is defined as constant now.